kornia_io/image.py

```diff
@@ -0,0 +1,85 @@
+from enum import Enum
+
+import cv2  # for reader
+import numpy as np
+
+from torch import Tensor
+import kornia as K
+from kornia.color import rgb_to_grayscale, bgr_to_grayscale
+
+# TODO: make it flexible
+__image_reader__ = cv2.imread
```

Comment:
is this wanted dependency?

Reply:
that's a good question. I would like to make it somehow flexible so that anyone can choose it's backend. However, we possibly will support our own thing later :)